front/public/locales/fr/errors.json

```diff
@@ -117,8 +117,8 @@
     },
     "scenario": {
       "NotFound": "Scénario non trouvé",
-      "InfraNotFound":"Infrastructure {{infra_id}} non trouvée",
-      "TimetableNotFound":"Grille horaire '{{timetable_id}}' non trouvée"
```

Comment:
Is it intended that these 2 keys are not translated in the `en` version (I know it's not related to this PR, but this change just made me check). Feel free to discard and/or send it to someone else for an answer/fix.

Reply:
We've removed the two errors (on the editoast side) and they weren't translated into English. I think someone forgot to do it on a previous PR.